backend/Wheelingful.API/Extensions/Endpoints/ChapterReaderExtension.cs

```diff
@@ -31,7 +31,7 @@ public static void MapChapterReaderApi(this IEndpointRouteBuilder endpoints)
             return TypedResults.Ok(result);
         });
 
-        chapterReaderGroup.MapGet("/book/{bookId}/chapter/{chapterId}", 
+        chapterReaderGroup.MapGet("/{chapterId}", 
```

Comment:
is chapterId unique in the whole system?

Reply:
Yes, that's right. It's ID does not depend on the Book entity.